test/local_test.go

```diff
@@ -20,7 +20,7 @@ func (s *CLITestSuite) TestLocalKafka() {
 		s.runIntegrationTest(tt)
 	}
 
-	time.Sleep(60 * time.Second)
+	time.Sleep(25 * time.Second)
```

Comment:
Would it make more sense to retry the connection every 10s? I've never seen this issue in integration tests before though... is it because we're using Docker?

Reply:
I don't think we should be using Docker in integration tests if we are... we should either mock it or move it to system tests. We shouldn't expect anyone to have to run Docker as a prerequisite to running `make test`... `make test` should just work on its own.

Reply:
it's running a real container so it needed a little time to set up the connection... 
you have a good point. For stablizing the tests for now i'll merge this pr first, and work on a mock for docker.